lib/cmds/ocr_cmds/get-suggestions.js

```diff
@@ -2,8 +2,33 @@
 
 const querystring = require('querystring');
 const moment = require('moment');
+const _find = require('lodash/find');
 const { get } = require('../../api');
+const { get: fhirGet, getAccount } = require('../../fhir');
 const print = require('../../print');
+const sleep = require('../../sleep');
+
+const waitSeconds = 900;
```

Comment:
This sets the timeout at a minimum of 15 minutes. The request also takes some time so it will end up being more than 15 minutes.